imagine/index.js

```diff
@@ -116,3 +113,8 @@ async function handleRequest(request) {
     });
   }
 }
+
+/* eslint-disable-next-line no-restricted-globals -- This is just how these webworkers are executed. Cannot change it. */
+addEventListener("fetch", (event) => {
+  event.respondWith(handleRequest(event.request));
+});
```

Comment:
To satisfy our eslint rules, this must move down in the file. Does not change the logic though.

Reply:
Please put that folder into the ignore list, I'm moving this code out of the repo

Reply:
Done.